### PR TITLE
chore: Update remix and netlify

### PR DESCRIPTION
**Pull request by istarkov**
## Description

fixes https://discord.com/channels/955905230107738152/1292985076702908500/1292985076702908500

tested deploy, builder, seems like all is fine

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/package.json [hunk 1]
```diff
@@ -43,14 +43,14 @@
...
-    "@remix-run/server-runtime": "^2.11.0",
+    "@remix-run/node": "^2.12.1",
+    "@remix-run/react": "^2.12.1",
+    "@remix-run/serve": "^2.12.1",
+    "@remix-run/server-runtime": "^2.12.1",
     "@trpc/client": "^10.45.2",
     "@trpc/server": "^10.45.2",
     "@tsndr/cloudflare-worker-jwt": "^2.5.3",
-    "@vercel/remix": "2.11.0",
+    "@vercel/remix": "2.12.0",
```
Comment (TrySound) [inline apps/builder/package.json:53]: no 2.12.1?

File: patches/@remix-run__dev.patch [hunk 2]
```diff
@@ -1,11 +1,11 @@
 diff --git a/dist/vite/node-adapter.js b/dist/vite/node-adapter.js
```
Comment (TrySound) [inline patches/@remix-run__dev.patch:1]: wow works without version???

File: apps/builder/package.json (hunk 1 above)
Comment (istarkov) [reply apps/builder/package.json:53]: no

File: patches/@remix-run__dev.patch (hunk 2 above)
Comment (istarkov) [reply patches/@remix-run__dev.patch:1]: Did same as before, seems like

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -16953,11 +16979,23 @@ snapshots:
       react-dom: 18.3.0-canary-14898b6a9-20240318(react@18.3.0-canary-14898b6a9-20240318)
       react-router: 6.26.0(react@18.3.0-canary-14898b6a9-20240318)
 
+  react-router-dom@6.26.2(react-dom@18.3.0-canary-14898b6a9-20240318(react@18.3.0-canary-14898b6a9-20240318))(react@18.3.0-canary-14898b6a9-20240318):
```
Comment (TrySound) [inline pnpm-lock.yaml:16982]: need to bump react-router-dom as well

**Review by TrySound — APPROVED** [review]

**Pull request merged**